sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -1011,9 +1011,9 @@ architecture::intel_gpu_dg2_g11, architecture::intel_gpu_dg2_g12`
 `architecture::intel_gpu_pvc`
 |8| `architecture::intel_gpu_dg2_g10,
 architecture::intel_gpu_dg2_g11, architecture::intel_gpu_dg2_g12`
-.4+|  `matrix_type::bf16`       .4+|  `matrix_type::bf16`   .4+|
-`matrix_type::fp32`  | 16 | 16 | 16 .3+|`architecture::intel_gpu_pvc`
-|32 | 64 | 16
+.5+|  `matrix_type::bf16`       .5+|  `matrix_type::bf16`   .5+|
+`matrix_type::fp32`  | 16 | 16 | 16 .4+|`architecture::intel_gpu_pvc`
+| 1 | 64 | 16 | 32 | 64 | 16
```

Comment:
The PR description says that you added 1x64x32.  However, it looks here like you added 1x64x16.  Which is right?

Just checking ... you added this only for PVC, right?

Reply:
Sorry, typo in PR description. Fixed and clarified that it is PVC only. This is correct.